pages/tests/[testsByRunId].tsx

```diff
@@ -67,20 +67,27 @@ type Props = {
   test_history: SuiteAndTest[]
 }
 
-function Tests(props: Props) {
-  const classes = useStyles(props)
+function Tests({ test_history }) {
+  const [history] = test_history
```

Comment:
Cristian, in theory all of this is redundant now and you can keep the old way of having props. test_run_data should be there already, inside props, so no need for this change (in theory)

Reply:
I applied object/array destructuring there so we can write less.

Reply:
okay, good idea!
Maybe we'll need to put a comment there now, that `if you want to use any other test_history property, please add in to that array`

Reply:
Actually, if you load a page that still doesn't have tests, you will end up with `is not an iterable` error, I guess we could put some validation on it @cristianmoisa 

<img width="1165" alt="Screenshot 2020-08-10 at 12 07 45" src="https://user-images.githubusercontent.com/14996980/89776949-bae1d000-db02-11ea-99fa-bcb0bd320bdb.png">